ts/Data/DataTableOptions.ts

```diff
@@ -40,7 +49,9 @@ export interface DataTableOptions {
 }
 
 
-export type DataTableValue = (boolean|null|number|string|undefined);
+export type DataTablePrimitiveValue = (boolean|null|number|string|undefined);
+
+export type DataTableValue = DataTablePrimitiveValue|DataTablePrimitiveValue[];
```

Comment:
Same question here. We have
```ts
CellType = BasicCellType|Array<BasicCellType>
```
in DataTable, but it does not get exposed further. I suggest to remove array as a cell type until we have a practical use case for it. It is not good for performance and produces management overhead. It is also a significant breaking change, independently whether we be consistent with typing or not. Every conditional check of types where last possible type is in the `else` section will not work anymore, when we introduce array.

Reply:
The only practical use for now is sparklines in Grid, but it's quite important. Do you have any suggestions for another way we can define data in cells for them?

Reply:
If this is a definite no-go, we can alternatively store the data as JSON string in cells and validate & parse it at the DataGrid level, on sparkline render.

Reply:
Yes, in the beginning we said that we go with JSON string, because otherwise you will always have to add new types for new cell content, if some structure has to be supported or extended.